specs/source.go

```diff
@@ -53,6 +54,12 @@ func (s *Source) SetDefaults() {
 		s.Tables = []string{"*"}
 	}
 
+	if s.Concurrency != 0 {
+		// attempt to make a sensible backwards-compatible choice, but the CLI
+		// should raise a warning about this until concurrency is fully removed.
```

Comment:
also need to check if both are set... and not overwrite if so, just leave the SDK to warn? oof too complex

Reply:
Yeah that's probably fine...

Reply:
But then it will break without warning when we remove this block in a future release.

Reply:
No, neither `table_concurrency` nor `resource_concurrency` are required, and if we remove this block it will be because we completely removed the `concurrency` option, so users will get an error about an unrecognized option.

Reply:
By "block" I meant this `if s.Concurrency != 0 {` code block. Resolving anyway 🤷